BaseApp/TAB99998.txt

```diff
@@ -0,0 +1,37 @@
+OBJECT Table 99998 CameraRequest
+{
+  OBJECT-PROPERTIES
+  {
+    Date=17-10-20;
+    Time=16:16:37;
+    Modified=Yes;
+    Version List=;
+  }
+  PROPERTIES
+  {
+  }
+  FIELDS
+  {
+    { 1   ;   ;Quality             ;Integer        }
+    { 2   ;   ;AllowEdit           ;Boolean        }
+    { 3   ;   ;EncodingType        ;Option        ;OptionString=JPEG,PNG }
+    { 4   ;   ;MediaType           ;Option        ;OptionString=Picture,Video,AllMedia }
+    { 5   ;   ;SourceType          ;Option        ;OptionString=Camera,PhotoLibrary }
+    { 6   ;   ;TakenPicture        ;BLOB           }
+    { 7   ;   ;TakenPictureName    ;Text250        }
+  }
+  KEYS
+  {
+    {    ;Quality                                 ;Clustered=Yes }
```

Comment:
If you end up saving this you should have a better key. A GUID will do.

Reply:
Table removed